### PR TITLE
Pg replication/migration/read-only replicas

### PR DESCRIPTION
**Pull request by ftisiot**
Some content addressing   [#33](https://github.com/aiven/devportal/issues/33)
* Logical replication
* create-read-replicas
* migrate-cloud-provider-region

### CODE REVIEW

File: docs/products/postgresql/howto/setup-logical-replication.rst [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+    When creating a publication entry, the ``publish`` parameter defines the operations to transfer. In the above example, all the ``INSERT``, ``UPDATE`` or ``DELETE`` operations will be transferred.
+
+3. PostgreSQL’s logical replication doesn’t copy table definitions, that can be extracted from the ``origin_database`` with ``pg_dump`` and included in a ``origin-database-schema.sql`` file with::
+
+    pg_dump --schema-only --no-publications \
+    SRC_CONN_URI                            \
+    -t test_table -t test_table_2 -t test_table_3 > origin-database-schema.sql
+
+4. The default ``postgres`` superuser :doc:`is not available in Aiven for PostgreSQL <../concepts/dba-tasks-pg>`, thus the user defined in the ``ALTER TABLE`` invocations within ``origin-database-schema.sql`` must be changed from ``postgres`` to the default ``avnadmin``.
```
Comment (lornajane) [inline docs/products/postgresql/howto/setup-logical-replication.rst:73]: So do we need to show people how to do this? Or do they need to take care about the user if they run an ALTER TABLE command on the source database? I'm not sure if this belongs here in the setup steps.

File: docs/products/postgresql/howto/setup-logical-replication.rst [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+The command output is like::
+
+        slot_name   │ restart_lsn
+     ───────────────┼─────────────
+      pghoard_local │ 6E/16000000
+           debezium | 5B/8B0
+     (2 rows)
+
+2. Compare the ``restart_lsn`` values between the replication slot in analysis (``debezium`` in the above example) and ``pghoard_local``: the hexadecimal difference between the them states how many write-ahead-logging (WAL) entries are waiting for the target ``debezium`` connector to catch up.
```
Comment (lornajane) [inline docs/products/postgresql/howto/setup-logical-replication.rst:153]: I'm really confused. We didn't mention Debezium before, is it just an example here?

File: docs/products/postgresql/howto/setup-logical-replication.rst [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+.. Note::
+    In the above example the difference is 0x6E - 0x5B = 19 entries
+
+
+3. If, after assessing the lag, the ``debezium`` connector results lagging or inactive:
+
+* If the ``debezium`` connector is still in use, a recommended approach is to restart the process and verify if it solves the problem.
```
Comment (lornajane) [inline docs/products/postgresql/howto/setup-logical-replication.rst:161]: How can the process be restarted?

**Review by lornajane — COMMENTED** [review]
I had some questions but this is pretty close to ready, IMO. Some language tweaks incoming in a commit as well

File: docs/products/postgresql/howto/setup-logical-replication.rst (hunk 1 above)
Comment (ftisiot) [reply docs/products/postgresql/howto/setup-logical-replication.rst:72]: It was in the original set of tasks in the doc and I was questioning myself as well. I'll remove it

File: docs/products/postgresql/howto/setup-logical-replication.rst (hunk 2 above)
Comment (ftisiot) [reply docs/products/postgresql/howto/setup-logical-replication.rst:152]: I removed the debezium reference and added the `dest_slot` naming in line with the example above

File: docs/products/postgresql/howto/setup-logical-replication.rst (hunk 3 above)
Comment (ftisiot) [reply docs/products/postgresql/howto/setup-logical-replication.rst:160]: Added a line on how to disable/enable it using aiven_extras

**Review by lornajane — APPROVED** [review]
Thanks! This looks good

**Pull request merged**